tests/integration/goldens/asset/google/cloud/asset_v1/services/asset_service/async_client.py

```diff
@@ -210,6 +210,31 @@ async def export_assets(self,
         the export operation result. For regular-size resource parent,
         the export operation usually finishes within 5 minutes.
 
+
+        .. code-block::
+        from google.cloud import asset_v1
```

Comment:
This indentation is a little unfortunate, but we can probably ignore it.

Reply:
Hrrm I didn't notice this earlier, I'll try poking at the template to see where this is coming from.